Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -67,14 +70,18 @@ - (void)viewDidLoad
                 }
             }
             
-            self.eventsDictionary = theEvents;
+            self.filteredEventsDictionary = theEvents; //Changed to filteredEventsDictionary
             // Sort the keys by date
             NSArray *keys = [theEvents allKeys];
             self.sortedDateKeys =  [keys sortedArrayUsingComparator: ^(NSString *d1, NSString *d2) {
                 NSDate *date1 = [NSDate dateFromString:d1];
                 NSDate *date2 = [NSDate dateFromString:d2];
                 return [date1 compare:date2];
             }];
+            
+            [self filterContentForSearchText:@"" scope:
```

Comment:
These lines seem very similar. Is there a way you could create a function? 

`Don't Repeat Yourself`